LAYOUT_STYLE_GUIDE.md

```diff
@@ -28,6 +28,7 @@ TOP uses Markdown for the layout and formatting of lesson and project files to g
 1.  [Links](#links)
 1.  [Images](#images)
 1.  [Codepen Embeds](#codepen-embeds)
+1.  [English](#english)
```

Comment:
Maybe instead of "English", we could use "Language" or something to that effect? Not sure if "Language" is the right one, but something along those lines is on the tip of my tongue. WDYT?

Reply:
> Maybe instead of "English", we could use "Language" or something to that effect? Not sure if "Language" is the right one, but something along those lines is on the tip of my tongue. WDYT?

@thatblindgeye My concern on just using "Language" as the section title would be a misinterpretation that this references programming languages or that it refers to appropriate/inappropriate language. 

Some other suggestions:

- "English language"

- "English writing style"

- "American English"

Reply:
I do like "English writing style", or some form of that like "American English writing style", "Preferred language in writing style", etc.

Reply:
@thatblindgeye Thanks, I updated the section title to "English Writing Style"